app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -6,6 +6,9 @@ extends 'katello/api/v2/common/timestamps'
 attributes :content_type
 attributes :docker_upstream_name
 attributes :unprotected, :full_path, :checksum_type, :container_repository_name
+if (@object && @object.yum?) || (@resource.is_a?(Katello::Repository) && @resource.yum?)
```

Comment:
@ehelms @cfouant had to fix this as `@resource` could be [defined as something other than Repository](https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/content_view_filters/show.json.rabl). Also, `@object` isn't defined when the controller renders this directly. Are you cool with this?